cluster/gce/config-default.sh

```diff
@@ -27,7 +27,7 @@ MINION_DISK_SIZE=100GB
 # for containervm image, so that gcloud will expand this
 # to the latest supported image.
 IMAGE=container-vm-v20150317
-IMAGE_PROJECT=google-containers
+IMAGE_PROJECT=google_containers
```

Comment:
Please don't change this. I can confirm that the underscore to dash aliasing is a special case to support old docker versions, as underscores aren't valid in project names.
```
gcloud compute instance-templates create test --image-project=google_containers
NAME MACHINE_TYPE CREATION_TIMESTAMP
ERROR: (gcloud.compute.instance-templates.create) Could not fetch image resource:
 - Invalid value 'google_containers'. Values must match the following regular expression: '(?:(?:[-a-z0-9]{1,63}\.)*(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?):)?(?:[0-9]{1,19}|(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?))'
```